### ISSUE TITLE
For each task there should be a task id generated. 

### ISSUE THREAD

**Issue by SAHARIPRASAD-2907** [mayank1513/vscode-extension-trello-kanban-board#37]
It will be really great if each task has a task id generated. It will be really helpful while write creating git commits and pull requests.

**Comment by mayank1513** [comment]
Every task already has a unique ID generated.

**Comment by mayank1513** [comment]
Can you please share more details as I am not able to understand your difficulty

**Comment by SAHARIPRASAD-2907** [comment]
Yes, I saw the .tkb file that contains Id for each and every task. Instead of having a task if like `task-8FtK0Q73PZNBQC5m35hRh` it will be really helpful if we can have a task if like 92234. based on the workspace we created. This feature will help us writing commit messages like 
T92235: Added files for project abc and after merging the pull request we can easily check the documentation based on the task id.
